charts/sscs-tribunals-api/values.preview.template.yaml

```diff
@@ -75,6 +75,13 @@ java:
     DOCUMENTS_WELSH_DIRECTION_ISSUED: TB-SCS-GNO-ENG-directions-notice-v2.docx
     BUNDLE_ENGLISH_CONFIG: sscs-ib-bundle-config.yaml
     BUNDLE_EDITED_CONFIG: sscs-ib-bundle-edited-config.yaml
+    NOTIFICATION_ENGLISH_ISSUEFINALDECISION_APPELLANT_DOCMOSISID: TB-SCS-GNO-ENG-00454-v2.docx
+    NOTIFICATION_ENGLISH_ISSUEFINALDECISION_APPOINTEE_DOCMOSISID: TB-SCS-GNO-ENG-00454-v2.docx
```

Comment:
how come the appointee letter is being updated given that we won't have appointees in IBA?